src/packager.ts

```diff
@@ -238,3 +241,12 @@ const objectExists = async (client: S3Client, bucket: string, key: string) => {
     throw error;
   }
 };
+
+const clearTempFolders = async () => {
+  const tempFolders = await fs.readdir(os.tmpdir());
+  tempFolders
+    .filter((f) => f.startsWith("temp-"))
```

Comment:
question: should this also include the `recovery-temp` directories?
question: given that it appears the `/tmp` folder is shared between instances of the Lambda, is it save to remove all directories, as other Lambdas might be using them?

suggestion: instead of deleting all the folders, 1) put the `recoveryTempDir` underneath `tempDir` 2) delete `tempDir` at the end of your processing (as you're doing with the `archiveRoot`)

Reply:
You could also delete the `recoveryTempDir` separately if that's easier than refactoring everything.